src/app/components/nav/dash-footer/dash-footer.component.html

```diff
@@ -0,0 +1,37 @@
+<footer class="ev-md-container dashboard-footer ev-dark-bg">
```

Comment:
I had reviewed the same file in #158!
Should we close/merge that first?

Reply:
I think we should close that PR